js/test/capitalize.test.js

```diff
@@ -0,0 +1,13 @@
+import capitalize from '../capitalize';
+
+test('It should capitalize the passed in string', () => {
+  expect(capitalize('test')).toMatch('Test');
+});
+
+test('It should only affect the first character', () => {
+  expect(capitalize('test').slice(1)).toMatch('est');
```

Comment:
- [ ] [OPTIONAL] Again, make sure to add more than one valid test case for the next project you add tests to.